pylint/testutils/functional/lint_module_output_update.py

```diff
@@ -22,7 +22,9 @@ class TestDialect(csv.excel):
         delimiter = ":"
         lineterminator = "\n"
 
-    csv.register_dialect("test", TestDialect)
+    # TestDialect inherit from csv.excel, which inherit from Dialect
+    # probably something wrong in csv typing
```

Comment:
I already fixed this in typeshed, sorry I didn't follow up with an ignore :-)

Reply:
Fixing the collections.abc bug was the priority back then :) !